tools/spec_template.rb

```diff
@@ -20,11 +20,13 @@
 class SpecTemplate
   attr_accessor   :version
   attr_accessor   :dependencies
+  attr_accessor   :required_helper_version
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.